lib/createForm.js

```diff
@@ -145,6 +141,35 @@ export const createForm = config => {
     util.update(touched, field, value);
   }
 
+  function isInitialValuesValid() {
+    if (Object.keys(initialValues).length < 1) {
+      const provided = JSON.stringify(initialValues);
+
+      console.warn(
+        `createForm requires initialValues to be a non empty object or array, provided ${provided}`
+      );
+
+      return false;
+    }
+
+    return true;
+  }
+
+  /**
+   * Update the initial values and reset form. Used to dynamically display new form values
+   */
+  function updateInitialValues(newValues, resetForm = true) {
```

Comment:
Should resetForm not always be the case? When does the user want to update the initialValues but not reset the form?

Reply:
To be honest, I don't remember what I had in mind when I did it!
**I removed it**...if it's really that important I'm sure we will run into it in the future ;)